Doesn't look for new version on inte to avoid privacy issuesIndex: lshw-02.14/src/core/version.cc
===================================================================
--- a/src/core/version.cc
+++ b/src/core/version.cc
@@ -112,6 +112,8 @@
 {
   static char *latest = NULL;
 
+  return NULL;
+
   if(!latest)
     latest = txtquery(PACKAGE, "ezix.org", NULL);
 
